glean_parser/__main__.py

```diff
@@ -74,7 +80,7 @@ def translate(input, format, output, option, allow_reserved):
     "--schema",
     "-s",
     type=str,
-    default=validate_ping.PING_SCHEMA_DEFAULT_URL,
+    default=PING_SCHEMA_DEFAULT_URL,
     nargs=1,
     required=False,
```

Comment:
I think we can also make this `required=True` (and remove the default).  That will change the CLI, so we'll have to confirm it doesn't break things, but we should be able to know when we file a PR to update `glean_parser` in `glean` before merging it, so low risk...